content/en/docs/concepts/extend-kubernetes/compute-storage-net/resource-drivers.md

```diff
@@ -0,0 +1,230 @@
+---
+title: DRA Resource Drivers
+description: Resource drivers provide non-trivial allocation logic and management for devices or resources that require vendor-specific or just complex setup, such as GPUs, NICs, FPGAs, etc.
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
```

Comment:
Is the specified feature state _(Kubernetes v1.26 [alpha])_  accurate as mentioned here? I noticed on another page ([Dynamic Resource Allocation page](https://kubernetes.io//docs/concepts/scheduling-eviction/dynamic-resource-allocation/)), which is also referenced in the content below, that the state is indicated as _(Kubernetes v1.27 [alpha])_.

Reply:
The feature was in fact added in 1.26:
```bash
$ git log v1.26.0 | grep -B4 "kube features: add DynamicResourceAllocation"
commit 155d49813f131a2beb6aeef71270f94b6f5fcf18
Author: Patrick Ohly <patrick.ohly@intel.com>
Date:   Tue Mar 22 16:53:58 2022 +0100

    kube features: add DynamicResourceAllocation
```
But, 1.27 had backwards-incompatible API change, I guess that was the reason to name it as 1.27 in the other page. I'll make it 1.27 here too for consistency.

Reply:
Force-pushed squashed commmits.

Reply:
`{{< feature-state for_k8s_version="v1.26" state="alpha" >}}` sounds correct. I'm inferring that the first alpha release was **v1.26**, even if there have been changes since.

https://kubernetes.io//docs/concepts/scheduling-eviction/dynamic-resource-allocation/ is IMO incorrect.